src/partition.rs

```diff
@@ -670,8 +670,13 @@ pub fn get_intra_edges<T: Pixel>(
         partition_bo.0.x > 0
       };
 
-    let right_available = x + tx_size.width() < dst.rect().width;
-    let bottom_available = y + tx_size.height() < dst.rect().height;
+    let rect_w =
```

Comment:
The rect_w, rect_h are derived from coded tile size. I.e. exclude the out of frame region because dst.rect().width|height now include out of frame region to access pixels belong to a block on frame boundary.

Reply:
> The rect_w, rect_h are derived from coded tile size. I.e. exclude the out of frame region because dst.rect().width|height now include out of frame region to access pixels belong to a block on frame boundary.

I might want to add member functions under rect of PlaneRegion, which returns these two values.